index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
design a favicon for our project

Reply:
![icon-512](https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/88591087/017ecb83-5cc8-4708-8a58-95957dbee0ca)

how this look like ?

Reply:
Looks great @wayneleon1 💪

Reply:
is it possible to make it transparent?

Reply:
> is it possible to make it transparent?
yh , let me try

Reply:
![icon-512-removebg-preview](https://github.com/atlp-rwanda/dynamites-ecomm-fe/assets/88591087/f205dfc6-533c-4ca4-b927-d28d735f0bc8)

Reply:
like that one